### DISCUSSION TITLE
Setup custom stlite sharing

### DISCUSSION THREAD

**Discussion by franciscobmacedo** [whitphx/stlite#552] [Q&A] [answered]
First of all, thanks for the great implementation.

I would like to run my own  instance of the [edit platform](https://edit.share.stlite.net/). I can't seem to find how to properly set it up locally. I tried to run `npm run install` within `packages/sharing` but I'm not sure how to install the other packages. For example, `@stlite/kernel` doesn't seem to be properly installed. Maybe I'm missing something, but is there some documentation for this?

Thanks!

**Comment by franciscobmacedo** [comment, marked as answer]
Nevermind, I just noticed the `DEVELOPMENT.md`file in the root directory 🤦

**Reply by whitphx** [reply]
Please let me know if `DEVELOPMENT.md` is out-dated!

**Reply by franciscobmacedo** [reply]
Actually, I think some things are missing. After running `make init` I also had to run `make streamlit-wheel` and `make stlite-server-wheel`before I could use the `sharing` and `sharing-editor` services. I can make a PR to update DEVELOPMENT.md if you want :)

**Reply by whitphx** [reply]
Thank you! Yes, it would be so helpful if you do it when you have time :D
